src/solvers/flattening/boolbv_quantifier.cpp

```diff
@@ -87,8 +87,10 @@ get_quantifier_var_max(const exprt &var_expr, const exprt &quantifier_expr)
         continue;
       if(expr_eq(var_expr, x.op0()) && x.op1().id()==ID_constant)
       {
-        exprt over_expr=x.op1();
+        const constant_exprt &over_expr = to_constant_expr(x.op1());
```

Comment:
:pick: While in the area, you might like to tidy the if to not duplicate the checking the id by using `try_dynamic_cast`

Reply:
or the version of `numeric_cast` which returns an optional